cpp/test/assembly/test.cpp

```diff
@@ -24,14 +24,19 @@
 #include <typedb.hpp>
 
 int main() {
-    TypeDB::Driver driver = TypeDB::Driver::coreDriver("127.0.0.1:1729");
-    std::string dbName = "testAssembly";
-    if (driver.databases.contains(dbName)) {
-        driver.databases.get("testAssembly").drop();
+    try {
+        TypeDB::Driver driver = TypeDB::Driver::coreDriver("127.0.0.1:1729");
+        std::string dbName = "testAssembly";
+        if (driver.databases.contains(dbName)) {
+            driver.databases.get("testAssembly").drop();
+        }
+        if (driver.databases.contains(dbName)) return 1;
+        driver.databases.create(dbName);
+        if (!driver.databases.contains(dbName)) return 1;
+        std::cout << "Success!" << std::endl;
+    } catch (TypeDB::DriverException e ) {
+        std::cout << "Caught TypeDB::DriverException: " << e.code() << "\n" << e.message()  << std::endl;
+        return 2;
```

Comment:
Catching this exception give us a helpful message, so we know it's not the assembly that failed but something earlier.